pom.xml

```diff
@@ -140,10 +140,10 @@
     <guava.version>22.0</guava.version>
     <groovy.version>2.4.21</groovy.version>
     <h2database.version>2.2.220</h2database.version>
-    <hadoop.version>3.3.6</hadoop.version>
+    <hadoop.version>3.4.1</hadoop.version>
     <hadoop.bin.path>${basedir}/${hive.path.to.root}/testutils/hadoop</hadoop.bin.path>
     <hamcrest.version>1.3</hamcrest.version>
-    <hbase.version>2.5.6-hadoop3</hbase.version>
+    <hbase.version>2.5.10-hadoop3</hbase.version>
```

Comment:
The HBase upgrade can be split into another ticket unless it can't go ahead without Hadoop upgade. If it is the case maybe then we can change the title to show that